website/events/api/v1/serializers/events/retrieve.py

```diff
@@ -58,12 +58,10 @@ def _description(self, instance):
         return strip_spaces_between_tags(bleach(instance.description))
 
     def _num_participants(self, instance):
-        if (
-            instance.max_participants
-            and instance.participants.count() > instance.max_participants
-        ):
+        p_count = instance.participants.count()
```

Comment:
```suggestion
        participant_count = instance.participants.count()
```

Easier to understand if the names are descriptive.